openedx/core/djangoapps/oauth_dispatch/dot_overrides/views.py

```diff
@@ -44,6 +44,7 @@ def get(self, request, *args, **kwargs):
             # at this point we know an Application instance with such client_id exists in the database
             application = get_application_model().objects.get(client_id=credentials['client_id'])
             kwargs['application'] = application
+            kwargs['organization_list'] = [org.short_name for org in application.organizations.all()]
```

Comment:
I changed around the data models slightly in the latest code. You want this to be:
```
kwargs['organization_list'] = [f.organization.short_name for f in application.org_filters.all()]
```